### PR TITLE
cn-pr-trans-index

### PR DESCRIPTION
**Pull request by markthink**
@zacharysarah  This adds the links to the KubeCon China site.

### CODE REVIEW

**Comment by dankohn** [comment]
@zacharysarah @kbarnard10 I had requested a quick turnaround update to the Chinese homepage to correctly link to the KubeCon Shanghai site. However, I'm not clear where that work should be occurring. However:

https://deploy-preview-9591--kubernetes-io-master-staging.netlify.com/cn/

Looks good to me.

File: content/cn/_index.html [hunk 1]
```diff
@@ -105,49 +108,34 @@ <h4><a href="/docs/concepts/workloads/controllers/jobs-run-to-completion/">批
...
-            <p>使用Kubernetes来重塑世界上最大的教育公司</p>
-            <a href="/case-studies/pearson/">阅读更多</a>
+            <p>使用 Kubernetes 支持快速决策应用程序</p>
+            <a href="/case-studies/capital-one">阅读更多</a>
         </div>
         <div>
-            <p>Kubernetes at Box: 微服务的最大优势</p>
-            <a href="https://blog.box.com/blog/kubernetes-box-microservices-maximum-velocity/">阅读更多</a>
+            <p>利用云原生态推动银行业创新</p>
+            <a href="/case-studies/ing">阅读更多</a>
```
Comment (chenrui333) [inline content/cn/_index.html]: 这个 href links 以后也需要update吧？

**Comment by zacharysarah** [comment]
@xiangpengzhao Can you review this for accuracy please? 

The k/kubernetes-docs-zh repo is definitely active for localization work, but putting KubeCon Shanghai information here for now makes sense. 

/assign @xiangpengzhao

**Comment by zacharysarah** [comment]
@xiangpengzhao 👋 ping

**Comment by zacharysarah** [comment]
@markthink Please feel free to reopen if you can provide an independent reviewer to verify the contents.

/cc @zhangxiaoyu-zidif @xiangpengzhao @tengqm

File: content/cn/_index.html [hunk 2]
```diff
@@ -4,80 +4,83 @@
...
-        
-        <h2>Kubernetes: 最后&hellip; 它是真正的云平台</h2>
-        <p>Box 的联合创始人和服务架构师 Sam Ghods 发表了热情洋溢的演讲，随着使用 Kubernetes，我们首次有一个通用接口，可以建立真正的部署工具。</p>
+        <h2>将 150+ 个微服务迁移到Kubernetes的挑战</h2>
+        <p>作者：Sarah Wells，运营和可靠性技术总监 - 金融时报</p>
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">观看视频</button>
+        <br>
+        <br>
+        <br>
+        <a href="https://www.lfasiallc.com/events/kubecon-cloudnativecon-china-2018/" button id= "desktopKCButton">2018年11月14日至15日在上海参加 KubeCon</a>
```
Comment (xiangpengzhao) [inline content/cn/_index.html]: 根据日程，应该是11月13日至15日

File: content/cn/_index.html [hunk 3]
```diff
@@ -4,80 +4,83 @@
...
+        <br>
+        <br>
+        <br>
+        <br>
+        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/" button id= "desktopKCButton">2018年12月11日至13日在西雅图参加 KubeCon</a>
```
Comment (xiangpengzhao) [inline content/cn/_index.html]: 根据日程，应该是12月10日至13日

File: content/cn/_index.html [hunk 4]
```diff
@@ -105,49 +108,34 @@ <h4><a href="/docs/concepts/workloads/controllers/jobs-run-to-completion/">批
...
-            <p>在 eBay 内部转移到 OpenStack 上的 Kubernetes 和容器</p>
-            <a href="https://www.nextplatform.com/2015/11/12/inside-ebays-shift-to-kubernetes-and-containers-atop-openstack/">阅读更多</a>
+            <p>Northwestern Mutual 的 Cloud Native</p>
+            <a href="/case-studies/northwestern-mutual/">阅读更多</a>
         </div>
         <div>
-            <p>将本地的集群迁移至 Kubernetes</p>
-            <a href="https://www.youtube.com/watch?v=6XGUTu3WhBw">观看视频</a>
+            <p>启动和扩展实验，变得简单</p>
+            <a href="/case-studies/openai/">观看视频</a>
```
Comment (xiangpengzhao) [inline content/cn/_index.html]: 英文版首页的最新案例已经变更，请同步更新

**Comment by xiangpengzhao** [comment]
@zacharysarah sorry for missing this thread :)
@markthink some inline comments, please update accordingly :)

**Comment by zacharysarah** [comment]
@xiangpengzhao Thanks for the feedback! 🙇 

@markthink Please feel free to re-open this PR when you're ready to address feedback. ☝️

**Pull request closed without merging**